Cargill/splinter-ui
# Simplify service section of circuit propose wizard

This PR reorganizes the circuit form modules and replaces the `ServiceCard` with a combination of a `ServiceTable` and `ServiceForm`.  These provide a more standard, clear way of creating services for a service.

To test:

```
docker-compose -f docker/docker-compose.yaml up --build
``` 

Navigate to [localhost:3031](http://localhost:3031) and create a user, if needed.

Add the "Alice" key via the profile sapling and activate.  The key can be obtained via:

```
docker-compose -f docker/docker-compose.yaml run generate-registry bash
cat /registry/alice.priv
cat /registry/alice.pub
```

Propose a circuit, select both nodes and click next.  You should see the following:
![Screen Shot 2020-08-13 at 11 48 09 AM](https://user-images.githubusercontent.com/282233/90162912-f7a80400-dd5a-11ea-8ecd-05d857043078.png)
Click the add service button and you will see:
![Screen Shot 2020-08-13 at 11 48 18 AM](https://user-images.githubusercontent.com/282233/90162939-02fb2f80-dd5b-11ea-94af-2699d8ab50fa.png)
After a service is added, it will appear in the table:
![Screen Shot 2020-08-13 at 11 49 51 AM](https://user-images.githubusercontent.com/282233/90163027-28883900-dd5b-11ea-8d8a-dc31970a1a49.png)
